<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Academy Engraved LET</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 236, 203);</string>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>2023 QS世界大學排名（美國）</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(200, 200, 203);</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioButton_all">
             <property name="font">
              <font>
               <family>Academy Engraved LET</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>QS 世界大學總排名</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_cate">
             <property name="font">
              <font>
               <family>Academy Engraved LET</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>QS 世界大學學科排名</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout" rowminimumheight="1,2">
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_all">
            <property name="font">
             <font>
              <family>Bodoni 72</family>
             </font>
            </property>
            <item>
             <property name="text">
              <string>Overall</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Academic Reputation</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Employer Reputation</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Faculty Student</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Citations per Faculty</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Internatonal Faculty</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>International Students</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>International Research Network</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Employment Outcomes</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Academy Engraved LET</family>
              <pointsize>13</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(200, 200, 203);</string>
            </property>
            <property name="text">
             <string>領域選擇：</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox_region">
            <property name="font">
             <font>
              <family>Bodoni 72</family>
             </font>
            </property>
            <item>
             <property name="text">
              <string>Accounting</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Agriculture</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Anatomy</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Anthropology</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Archaeology</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Architecture</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Art Design</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Arts Humanities</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Biological Sciences</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Business</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Chemistry</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Classics</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Communication</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Computer Science</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Data Science</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Dentistry</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Development Studies</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Earth Marine Sciences</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Economics Econometrics</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Education</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Engineering Chemical</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Engineering Civil</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Engineering Electrical</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Engineering Mechanical</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Engineering Mineral</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Engineering Petroleum</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Engineering Technology</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>English Language</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Environmental Sciences</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Geography</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Geology</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Geophysics</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>History of Art</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>History</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Hospitality</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Law</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Library</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Linguistics</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Life Sciences Medicine</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Marketing</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Materials Science</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mathematics</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Medicine</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Modern Languages</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Natural Sciences</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Nursing</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Performing Arts</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pharmacy</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Philosophy</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Physics</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Politics</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Psychology</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Social Policy</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Social Sciences Management</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sociology</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sports related Subjects</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Statistics</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Theology</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Veterinary Science</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <family>Academy Engraved LET</family>
                <pointsize>13</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(200, 200, 203);</string>
              </property>
              <property name="text">
               <string>學科排名依據</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Academy Engraved LET</family>
              <pointsize>13</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(200, 200, 203);</string>
            </property>
            <property name="text">
             <string>排名依據：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="comboBox_subject">
            <property name="font">
             <font>
              <family>Bodoni 72</family>
             </font>
            </property>
            <item>
             <property name="text">
              <string>Score</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Academic</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Employer</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Citations</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>H</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QPushButton" name="pushButton_loadcsv">
            <property name="font">
             <font>
              <family>Bodoni 72</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>下載csv</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1">
          <item>
           <widget class="PlotWidget" name="gView"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QWebEngineView" name="webEngineView" native="true">
              <property name="styleSheet">
               <string notr="true">background-image: 'star.jpeg'</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableView" name="tableView"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>118</width>
                <height>28</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_first">
              <property name="font">
               <font>
                <family>Academy Engraved LET</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: skyblue</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>rewind-button.png</normaloff>rewind-button.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_front">
              <property name="font">
               <font>
                <family>Academy Engraved LET</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: skyblue</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>previous.png</normaloff>previous.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_page"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_next">
              <property name="font">
               <font>
                <family>Academy Engraved LET</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: skyblue</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>next.png</normaloff>next.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_last">
              <property name="font">
               <font>
                <family>Academy Engraved LET</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: skyblue</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>forward-button.png</normaloff>forward-button.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>108</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>webengineview</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
